src/Process-Data/Get_Data.py

```diff
@@ -33,26 +36,34 @@
 count = 0
 year_count = 0
 
+con = sqlite3.connect("../../Data/teams.sqlite")
+
 for season1 in tqdm(season):
     for month1 in tqdm(month):
         if month1 == 1:
             count += 1
             end_year_pointer = year[count]
         for day1 in tqdm(days):
-            try:
-                general_data = get_json_data(
-                    url.format(month1, day1, begin_year_pointer, end_year_pointer, season1))
-                general_df = to_data_frame(general_data)
-                real_date = date(year=end_year_pointer, month=month1, day=day1) + timedelta(days=1)
-                general_df['Date'] = str(real_date)
-
-                directory2 = os.fsdecode('../../Team-Data')
-                x = str(real_date).split('-')
-
-                name = directory2 + '/' + '{}-{}-{}'.format(str(int(x[1])), str(int(x[2])), season1) + '.xlsx'
-                general_df.to_excel(name)
-            except:
+            if month1 == 10 and day1 < 19:
```

Comment:
Some seasons didn't start at 10. I think maybe the old way was better to more easily run. I still think this could be better.

Reply:
I was getting errors with the original code so I made some changes.  Didn't realize that some seasons didn't start in october.  I'll take a closer look at this.

Reply:
This code still works fine for seasons that don't start with 10 as far as I can tell.  Let me know if I'm missing something